x-pack/plugins/fleet/common/types/models/output.ts

```diff
@@ -26,6 +26,16 @@ export interface NewOutput {
     key?: string;
   } | null;
   proxy_id?: string | null;
+  disk_queue_enabled?: boolean;
+  disk_queue_path?: string;
+  disk_queue_max_size?: number;
+  disk_queue_encryption_enabled?: boolean;
+  disk_queue_compression_enabled?: boolean;
+  compression_level?: number;
+  loadbalance?: boolean;
+  mem_queue_events?: number;
+  queue_flush_timeout?: number;
+  max_batch_bytes?: number;
```

Comment:
What do you think of putting all this properties under a `shipper` one?

Reply:
I moved these properties under `shipper` and updated the saved object as well.